library/ecdsa.c

```diff
@@ -47,12 +47,38 @@
 #include "mbedtls/platform_util.h"
 #include "mbedtls/error.h"
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+#include "mbedtls/pk.h"
+#endif
+
 /* Parameter validation macros based on platform_util.h */
 #define ECDSA_VALIDATE_RET( cond )    \
     MBEDTLS_INTERNAL_VALIDATE_RET( cond, MBEDTLS_ERR_ECP_BAD_INPUT_DATA )
 #define ECDSA_VALIDATE( cond )        \
     MBEDTLS_INTERNAL_VALIDATE( cond )
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+int mbedtls_pk_ecp_psa_err_translate( psa_status_t status )
```

Comment:
This function should either be called `mbedtls_pk_xxx`, defined in `pk*.c` and declared in `pk*.h`; or called `mbedtls_ecp_xxx`, defined in `ecp.c` and declared in `ecp*.h`. Likewise for RSA.